content/en/Getting started/glossary.md

```diff
@@ -280,20 +280,20 @@ For pentests of a GraphQL API, Cobalt needs the number of [queries and mutations
 
 For more information, see https://graphql.org/learn/queries/
 
-## Gray Box Testing
+## Gray-Box Testing
 
 Where the pentester has limited knowledge of the internal details of the asset. Contrast
-with white box and black box testing.
+with [white-box](#white-box-testing) and [black-box testing](#black-box-testing).
 
-Also known as "translucent box testing."
+Also known as "translucent-box testing."
 
 ## Jump Box
 
 Also known as a jump host or a jump server, a jump box is a system (typically) on an internal
 network or a DMZ. Jump boxes are used to access and manage devices in a separate security zone.
 
 Where the pentester has limited knowledge of the internal details of the asset. Contrast
-with white box and black box testing.
+with [white-box](#white-box-testing) and [black-box testing](#black-box-testing).
 
 ## Known Vulnerability
 
```

Comment:
I've generally been using the term 'Documented' vulnerability over 'Known' because not everyone necessarily "knows" about the vuln, but it can be researched. Couldn't hurt to add an AKA perhaps?

Reply:
@TheInstaGrahame Yes, that's why we're also mentioning this:

> In Cobalt pentest reports, you may see this as a published or documented vulnerability.